src/eukyungbin/ManagementSys.java

```diff
@@ -0,0 +1,118 @@
+package eukyungbin;
+
+import jun.ManagementSystem;
+import jun.Member;
+
+import java.util.*;
+import java.util.function.Predicate;
+
+public class ManagementSys implements ManagementSystem {
+
+    HashMap<Integer, Member> map = new HashMap<>(); //멤버의 id와 Member객체를 갖는 map
+
+    @Override
+    public boolean addMember(String name, int age, int id) {
+        if(!map.containsKey(id)){
+            Member newMember = new Member(name, age, id); //새로운 Member객체 만들기
+            map.put(id, newMember); // 없는 id면 map에 추가하기
+            return true;
+
+        }
+
+        return false;
+    }
```

Comment:
먼저 과제 제출 수고하셨습니다!

개발은 사소하지만 좋은 습관이 자리잡혀 있는 개발자가 좋은 개발자라고 생각해요
코드를 작성하실때에는 "관례"라는것이 있어요
이런 좋은 습관들이 자리잡기에 가장 좋은때는 개발을 처음 배울때가 중요하다고 생각해요
몇가지 리뷰들을 살펴보시고 좋은 습관 만들어봐용

1. ``` return ``` 을 작성 하실때에는 코드의 가독성을 위한 관례로 주된 로직과의 줄바꿈은 관례에요
팀원들과의 정해진 코드 컨벤션이 있다면 이야기가 달라지지만 보편적으로는 그렇습니다

2. ``` if (...) { } ``` 처럼 조건문이나 반복문 뒤에 위치하는 소괄호들은 띄워주는것이 관례에요

3.  2번과 마찬가지로 클래스, 메서드, 조건문, 반복문을 박싱하는 중괄호들은 꼭 띄워주시는게 좋아요

```suggestion
    @Override
    public boolean addMember(String name, int age, int id) {
        if (!map.containsKey(id)) {
            Member newMember = new Member(name, age, id); //새로운 Member객체 만들기
            map.put(id, newMember); // 없는 id면 map에 추가하기
            
            return true;
        }

        return false;
    }
```